project-base/tests/ReadModelBundle/Functional/Twig/ImageExtensionTest.php

```diff
@@ -90,6 +92,36 @@ public function testGetImageHtml(): void
         libxml_clear_errors();
     }
 
+    public function testGetImageHtmlWithoutNativeLazyLoad(): void
+    {
+        $productId = 1;
+        $entityName = 'product';
+        $fileExtension = 'jpg';
```

Comment:
This is used more than once in this class - maybe it could be refactored to constants?

Reply:
Or maybe even better - creation of ImageView into method